tests/services/order_validation_service/test_inter_field_validators.py

```diff
@@ -0,0 +1,16 @@
+from cg.services.order_validation_service.models.order import Order
+from cg.services.order_validation_service.validators.inter_field_validators import (
+    validate_ticket_number_required_if_connected,
+)
+
+
+def test_ticket_is_required(valid_order: Order):
+    # GIVEN an order that should be connected to a ticket but is missing a ticket number
+    valid_order.connect_to_ticket = True
+    valid_order.ticket_number = None
+
+    # WHEN validating the order
+    errors = validate_ticket_number_required_if_connected(valid_order)
+
+    # THEN an error should be returned
+    assert errors
```

Comment:
we should assert the correct errors are raised

Reply:
https://github.com/Clinical-Genomics/cg/pull/3426/commits/51145d59787e792be6603a62ec6afc3a729bbf97